tests/Unit/Client/SerializerTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle\Tests\Unit\Client;
+
+use Answear\EcontBundle\Client\Serializer;
+use Answear\EcontBundle\Request\GetOfficesRequest;
+use Answear\EcontBundle\Request\Request;
+use PHPUnit\Framework\TestCase;
+
+class SerializerTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    /**
+     * @test
+     * @dataProvider provider
+     */
+    public function requestAuthentication(Request $request, string $expectedBody): void
```

Comment:
We're not testing authentication but rather body serialization

Reply:
Yeah, copied from another bundle :)